napari/utils/action_manager.py

```diff
@@ -198,11 +199,18 @@ def bind_button(
         delayed until the corresponding action is registered.
         """
```

Comment:
```suggestion
        delayed until the corresponding action is registered.
        
        Note: this method cannot be used with generator functions.
            see https://github.com/napari/napari/issues/4164 for details.
        """
```

Reply:
@tlambert03 added a note to the docstring, feel free to include or not, as you see fit

Reply:
Tyvm!